core/ir/disassemble_shared.c

```diff
@@ -1164,6 +1224,48 @@ sign_extend_immed(instr_t *instr, int srcnum, opnd_t *src)
     }
 }
 
+#define MAX_LENGTH_ALL_CATEGORIES_AS_STRING 57
```

Comment:
Perhaps this and the following two functions should be moved to a new file: core/ir/isa_regdeps/disassemble.c.

Reply:
+1, see comment above too.

Reply:
Done.